apps/web/components/PoolCard.tsx

```diff
@@ -24,60 +28,45 @@ export function PoolCard({
   enabled = true,
 }: StrategyQuery & { tokenGarden: TokenGarden | undefined }) {
   const pathname = usePathname();
-
   poolAmount = poolAmount || 0;
+  const poolType = config?.proposalType as number;
+
   return (
-    <div className="relative flex max-w-56 rounded-md bg-white shadow transition-all duration-150 ease-out ">
-      {!enabled && (
-        <div className="border2 absolute top-10 z-50 w-full bg-warning">
+    <Card href={`${pathname}/pool/${poolId}`}>
+      <header className="mb-4 flex w-full items-center justify-between">
+        <h4>Pool #{poolId}</h4>
+        <Badge poolType={poolType} />
+      </header>
+      <div className="mb-10 flex min-h-[60px] flex-col gap-2">
+        <Identifier
+          icon={<HandRaisedIcon />}
+          count={proposals.length}
+          label="proposals"
+        />
+        {poolType == 1 && (
```

Comment:
do we have a type for poolType? Maybe more clear to map a type to a number I think

Reply:
we have types yes, also it is used across app these logic so we can apply like you said instead